conf/empire/empire.yaml

```diff
@@ -180,3 +180,9 @@ stacks:
       GithubWebhooksSecret: ${empire_controller_github_webhooks_secret}
       GithubDeploymentsEnvironment: ${empire_controller_github_deployments_environment}
       TokenSecret: ${empire_controller_token_secret}
+      DockerImage: mhahn/empire:b7c43199429f2567741684be52a5701f56f20d56
+  - name: acmeApp
```

Comment:
Can you throw a comment in here detailing what this is, etc?  It might be worth actually commenting it out entirely by default, since people probably copy this file the first time they setup Empire.